src/server/page.ts

```diff
@@ -359,15 +361,19 @@ export class Page extends SdkObject {
     }), this._timeoutSettings.navigationTimeout(options));
   }
 
-  async emulateMedia(options: { media?: types.MediaType | null, colorScheme?: types.ColorScheme | null }) {
+  async emulateMedia(options: { media?: types.MediaType | null, colorScheme?: types.ColorScheme | null, reducedMotion?: types.ReducedMotion | null }) {
     if (options.media !== undefined)
```

Comment:
It is safe to drop these checks, client should take care of it. And if it does not, protocol validator will yell anyway.

Reply:
So which types instead?a

Reply:
Not sure what you mean with the last question.My point is that we can remove lines 365-370 in the new code, and all tests will still pass.